### DISCUSSION TITLE
Using skmultilearn to learn a semi-supervised model

### DISCUSSION THREAD

**Discussion by bundit786** [scikit-multilearn/scikit-multilearn#306] [Q&A]
Hi.
I'm working on a semi-supervised learning model with multi-label data. Is there any modification that we can learn a semi-supervised model? Within `sklearn.semi_supervised` there are methods like self-training, label propagation, and label spreading but they do not support multi-label data.

Thank you.
